src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -886,7 +890,12 @@ function discoverController(
   $scope.updateDataSource = Promise.method(function updateDataSource() {
     $scope.searchSource
       .setField('size', $scope.opts.sampleSize)
-      .setField('sort', getSort($state.sort, $scope.indexPattern))
+      .setField('sort', [
+        getSort($state.sort, $scope.indexPattern),
+        {
+          [tiebreakerField]: defaultSortOrder,
```

Comment:
No tiebreakerField (I've set the field to empty in Advanced Configuration) leads to the following sort `{"undefined":{"order":"desc","unmapped_type":"boolean"}`.